### DISCUSSION TITLE
Is there a way to use RTL-MP2 as standalone

### DISCUSSION THREAD

**Discussion by arun-chaubey** [The-OpenROAD-Project/OpenROAD#4119] [Q&A]
Hello,

Is there any way I can use just the RTL-MP2 stand alone on either a RTL design or a gate netlist ? Issue is, we use proprietary tools so do not have OpenDB database for our designs.

Thanks & Regards,
Arun

**Comment by maliberty** [comment]
You can just read in a DEF, run it, and write it a DEF
